pypit/ararc.py

```diff
@@ -79,7 +79,7 @@ def detect_lines(slf, det, msarc, censpec=None, MK_SATMASK=False):
         satmask = arcyarc.saturation_mask(msarc, slf._nonlinear[det-1])
         satsnd = arcyarc.order_saturation(satmask, ordcen, (ordwid+0.5).astype(np.int))
     else:
-        satsnd = np.zeros_like(ordcen)
+        satsnd = np.zeros_like(censpec)
```

Comment:
I think we want to keep this as it was, and instead, on line 123 have:

`tpixt, num = arcyarc.detections_sigma(yprep, yerr, np.zeros(censpec.shape[0], dtype=np.int), siglev/2.0, siglev)`

That should fix one of the bugs